client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -217,13 +280,68 @@ const PlanOnlyThankYou = ( {
 					/>
 				}
 				footerDetails={ footerDetails }
+				upsellProps={ upsellProps }
 			/>
 		</>
 	);
 };
 
-export default connect( null, {
-	errorNotice,
-	removeNotice,
-	successNotice,
-} )( PlanOnlyThankYou );
+function getBusinessPlanSlugToUpgrade( currentPlanSlug: string ) {
+	const currentPlan = getPlan( currentPlanSlug );
+	const planKey = findFirstSimilarPlanKey( PLAN_BUSINESS, { term: currentPlan?.term } );
+
+	// It's just for TS, since .find() can return undefined
+	if ( ! planKey ) {
+		return;
+	}
+
+	return getPlan( planKey );
+}
```

Comment:
Could you move that function next to `isMonthsOld()`? It's a bit odd to find it here - between the component and the `connect()` call.